src/cdk/overlay/position/connected-position.ts

```diff
@@ -73,6 +79,9 @@ export class ScrollingVisibility {
 
 /** The change event emitted by the strategy when a fallback position is used. */
 export class ConnectedOverlayPositionChange {
-  constructor(public connectionPair: ConnectionPositionPair,
-              @Optional() public scrollableViewProperties: ScrollingVisibility) {}
+  constructor(
+      /** The position used as a result of this change. */
+      public connectionPair: ConnectionPositionPair,
+      /** @docs-private */
```

Comment:
If this is `@docs-private` should it be `_scrollableViewProperties`

Reply:
Can't rename b/c someone in google is using it